doc/Language/5to6-nutshell.pod6

```diff
@@ -1878,4 +1878,4 @@ live on GitHub. An online converter may become available at some point.
 
 =end pod
 
-# vim: expandtab softtabstop=4 shiftwidth=4 ft=perl6
+# vim: expandtab softtabstop=4 shiftwidth=4 ft=raku
```

Comment:
This is the problem with massive PRs with many files: there might be one quirk and everything goes tumbling down. In this case, I'm not sure this would work (for those using vim). It might, however.